src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -505,19 +505,16 @@ protected void cleanTmpDir(Path tmpDir) {
     private void execute(CommandLine command, String stdin) {
         InputStream inputStream = new ByteArrayInputStream(stdin.getBytes());
 
+        class InfoLogOutputStream extends LogOutputStream {
+            @Override protected void processLine(String line, int level) {
+                logger.info(line);
+            }
+        };
 
         try (
-            LogOutputStream outputStream = new LogOutputStream() {
-                @Override protected void processLine(String line, int level) {
-                    logger.info(line);
-                }
-            };
-            LogOutputStream errorStream = new LogOutputStream() {
-                @Override protected void processLine(String line, int level) {
-                    logger.error(line);
-                }
-            }
-        ){
+            LogOutputStream outputStream = new InfoLogOutputStream();
```

Comment:
Is it possible to instantiate just one instance of InfoLogOutputStream, and then pass it to the setStreamHandler() for both output and error?

Unless there's an issue with concurrency? It doesn't look like the override of processLine is synchronized so I'm not sure it would make any difference?